index.js

```diff
@@ -35,6 +36,17 @@ class Code {
     return Toolbox;
   }
 
+  get CSS() {
```

Comment:
![Screenshot 2023-10-16 at 13 11 33](https://github.com/Juratbek/editorjs-code/assets/55176773/f77b4a0e-63d2-432e-a64c-f6da42e970de)
there should not be border for copy button

Reply:
the language selector is not being closed on click outside also. what is fixed in this PR?

Reply:
Did you build the plugin first?